### DISCUSSION TITLE
API for posting new feed to the global wall

### DISCUSSION THREAD

**Discussion by ankit-vkaps** [HPaulson/BitClout-API-Docs#6] [Q&A] [answered]
Hello,

Can you please provide API endpoint to post info or message on bitclout global wall, so we can connect it with other platforms to post messages directly via API?
![image](https://user-images.githubusercontent.com/75479262/114894070-ebb19c00-9e2b-11eb-8c69-dbc10ace68bf.png)

**Comment by HPaulson** [comment, marked as answer]
@jrawlings There's currently no API endpoint to accomplish such (for users, at least). Users cannot get their post on the global feed via the client (unless they have admin permissions), as this would make it so anyone could have all of their posts on global -- which of course isn't ideal.

**Comment by ankit-vkaps** [comment]
I actually need to post into the global feed

**Reply by HPaulson** [reply]
Yeah as said above, this isn't currently possible (and definitely won't be possible in the future). As of right now, posts can only be added to the global feed by:

A. Having an admin manually add it on BitClout.com through the post dropdown

B. Getting it in the algorithm (Note this is an assumption)


While there is an API endpoint to manually add posts to the global feed, however since only admins can authenticate the request using the NodeSecret, there's no value in me providing documentation on it for standard users.
